src/pal/src/include/pal/stackstring.hpp

```diff
@@ -5,7 +5,9 @@
 #ifndef __STACKSTRING_H_
 #define __STACKSTRING_H_
 
+#ifndef __STACKSTRING_H_STANDLONE_
 #include "pal/malloc.hpp"
```

Comment:
I have removed it, did not have to modify any other site, the malloc.hpp is brought in by other headers presumably